Content.Server/_L5/StaticField/StaticFieldSystem.cs

```diff
@@ -51,15 +51,16 @@ private void SetState(Entity<StaticFieldComponent> ent, bool state)
     {
         ent.Comp.Powered = state;
```

Comment:
Is this Powered field still needed?

Reply:
It's used in toggling the status, line 46